src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -80,22 +87,33 @@ public CommandBase rotateToAngle(Rotation2d angle, DoubleSupplier translateX, Do
 
     }
 
-    public CommandBase setSpeedCoef(double fac) {
+    public CommandBase setSpeedCoef(double fac, double rotationfac) {
         return runOnce(() -> {
             speedCoef = fac;
+            rotationCoef = rotationfac;
+
         });
     }
 
     private void move(final double xSpeed, final double ySpeed,
             final double rotation) {
 
         final Modifier deadband = Modifier.deadband(0.15);
+
+        double rotationInput = deadband.applyAsDouble(rotation);
+
+        if (Math.abs(rotationInput) < 0.1) {
```

Comment:
I'd love to have a comment here explaining what this does

Reply:
Joe said he'll do it later